native/acter/src/api/verification.rs

```diff
@@ -1151,7 +1155,7 @@ impl VerificationController {
                             evt.sender,
                         );
                         msg.set_content("keys".to_string(), evt.content.keys.to_string());
-                        let mac = serde_json::to_string(&evt.content.mac).unwrap();
+                        let mac = serde_json::to_string(&evt.content.mac).expect("couldn't convert mac to string");
```

Comment:
same logging as above instead of panic...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1245/commits/a183faf7571b2a83fa1b0d3e33d6a15d2fb4f03b